include/envoy/http/filter.h

```diff
@@ -164,8 +164,10 @@ class StreamDecoderFilterCallbacks : public virtual StreamFilterCallbacks {
   /**
    * Continue iterating through the filter chain with buffered headers and body data. This routine
    * can only be called if the filter has previously returned StopIteration from decodeHeaders()
-   * AND either StopIterationAndBuffer or StopIterationNoBuffer from each previous call to
-   * decodeData(). The connection manager will dispatch headers and any buffered body data to the
+   * AND one of StopIterationAndBuffer, StopIterationAndWatermark, or StopIterationNoBuffer
```

Comment:
Do you mind fixing this same issue in `continueEncoding()` below?

Reply:
Done. I was focused on the Decode path and didn't think to look at the Encode side too.